test/internet/test-dgram-multicast-multi-process.js

```diff
@@ -1,18 +1,115 @@
 'use strict';
-var common = require('../common'),
-    assert = require('assert'),
-    dgram = require('dgram'),
-    util = require('util'),
-    Buffer = require('buffer').Buffer,
-    fork = require('child_process').fork,
-    LOCAL_BROADCAST_HOST = '224.0.0.114',
-    TIMEOUT = common.platformTimeout(5000),
-    messages = [
-      new Buffer('First message to send'),
-      new Buffer('Second message to send'),
-      new Buffer('Third message to send'),
-      new Buffer('Fourth message to send')
-    ];
+const common = require('../common');
+const assert = require('assert');
+const dgram = require('dgram');
+const Buffer = require('buffer').Buffer;
+const fork = require('child_process').fork;
+const LOCAL_BROADCAST_HOST = '224.0.0.114';
+const TIMEOUT = common.platformTimeout(5000);
+const messages = [
+        new Buffer('First message to send'),
+        new Buffer('Second message to send'),
+        new Buffer('Third message to send'),
+        new Buffer('Fourth message to send')
+      ];
+
+// skip test in FreeBSD jails
+if (common.inFreeBSDJail) {
+  console.log('1..0 # Skipped: In a FreeBSD jail');
+  return;
+}
+
+function launchChildProcess(index) {
+  const worker = fork(process.argv[1], ['child']);
+  workers[worker.pid] = worker;
```

Comment:
Maybe hoist `workers` to the top while you're here, it arguably reads easier that way.